modules/bibauthorid/lib/bibauthorid_dbinterface.py

```diff
@@ -1785,7 +1789,7 @@ def remove_arxiv_papers_of_author(pid):
             'where tag=%s and personid=%s', ('arxiv_papers', pid))
 
 
-def _add_external_id_to_author(pid, ext_sys, ext_id):  # add_personID_external_id
+def add_external_id_to_author(pid, ext_sys, ext_id, avoid_id_duplication=True):   ### add_personID_external_id
```

Comment:
@petros-ioannidis again change of function name. Have you checked if it is ok elsewhere?